dds_web/commands.py

```diff
@@ -558,7 +558,9 @@ def delete_invites():
                             "Invite with created_at = 0000-00-00 00:00:00 deleted."
                         )
                     else:
-                        flask.current_app.logger.debug("Invite deleted.")
+                        flask.current_app.logger.debug(
+                            f"Invite deleted: {invite.email} (created at {invite.created_at})."
+                        )
```

Comment:
Looks good. 
I think we should add the following as well though: 
```suggestion
                        flask.current_app.logger.debug(
                            f"Invite deleted: {invite.email} (created at {invite.created_at})."
                        )
                    any_deleted  = True
```

And then you need to change these parts

**Line 549-550:**
```python
        invites: list = db.session.query(models.Invite).all()
        any_deleted: bool = False
```

**Line 571-575:**
```python
        if not any_deleted:
            flask.current_app.logger.info("No invites deleted.")
    except (OperationalError, SQLAlchemyError) as err:
        flask.current_app.logger.exception(err)
        raise
```